app/serializers/rest/instance_serializer.rb

```diff
@@ -24,6 +24,12 @@ def version
     Mastodon::Version.to_s
   end
 
+  def stats
+    { user_count:   User.confirmed.count,
```

Comment:
Most likely, but I have no clue how to do that. My Ruby (and Rails) knowledge is severely lacking, and have no idea how I could use `InstancePresenter` in the serializer, or how I could make it available to it.

I tried searching for examples, but none of them gave me enough clues :(